grunt/config/uglify.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (grunt, options) {
+    return {
+        libraries: {
```

Comment:
you need to add 
```js
options: {
    ie8: true
},
```
here or the minified version of the code won't load in IE8. We can take this out again once we move to v3 and drop IE8 support.